Libraries/Image/ImageStylePropTypes.js

```diff
@@ -32,6 +32,12 @@ var ImageStylePropTypes = {
   // It changes the color of all the non-transparent pixels to the tintColor
   tintColor: ColorPropType,
   opacity: ReactPropTypes.number,
+
+  // Android-Specific styles
```

Comment:
Can you verify that this is indeed required here? Looks like all the properties you're adding are already a part of `ViewStylePropTypes`. I prefer for the way of declaring props for the views to be consistent across View and Image (that would be, just expecting it to be set in `style`).

I think what you should do here is to actually just get rid of these few lines added. Then for `Image` component you can put `borderTopLeftRadius` and others as a part of `style` and it should work with no additional changes.

Also: Can you include a screenshot of "Border Radius" example from `ImageExample.js` to confirm there is no regression there. Note that in the aforementioned example `borderRadius` is set through `style` property.